pkg/allowdenylist/allowdenylist.go

```diff
@@ -107,31 +107,31 @@ func (l *WhiteBlackList) IsIncluded(item string) bool {
 		}
 	}
 
-	if l.isWhiteList {
+	if l.isAllowList {
 		return matched
 	}
 
 	return !matched
 }
 
 // IsExcluded returns if the given item is excluded.
-func (l *WhiteBlackList) IsExcluded(item string) bool {
+func (l *AllowDenyList) IsExcluded(item string) bool {
 	return !l.IsIncluded(item)
 }
 
-// Status returns the status of the WhiteBlackList that can e.g. be passed into
+// Status returns the status of the AllowDenyList that can e.g. be passed into
 // a logger.
-func (l *WhiteBlackList) Status() string {
+func (l *AllowDenyList) Status() string {
 	items := make([]string, 0, len(l.list))
 	for key := range l.list {
 		items = append(items, key)
 	}
 
-	if l.isWhiteList {
-		return "whitelisting the following items: " + strings.Join(items, ", ")
+	if l.isAllowList {
+		return "allowlisting the following items: " + strings.Join(items, ", ")
 	}
 
-	return "blacklisting the following items: " + strings.Join(items, ", ")
+	return "denylisting the following items: " + strings.Join(items, ", ")
```

Comment:
hmm I would reword all the user facing strings to make it more readable, wdyt of something like:
> Excluding the following lists that were on denylist: .....

Reply:
Done